vendor/modules.txt

```diff
@@ -743,7 +743,7 @@ github.com/sigma/go-inotify
 github.com/sirupsen/logrus
 # github.com/soheilhy/cmux v0.1.3 => github.com/soheilhy/cmux v0.1.3
 github.com/soheilhy/cmux
-# github.com/spf13/afero v0.0.0-20160816080757-b28a7effac97 => github.com/spf13/afero v0.0.0-20160816080757-b28a7effac97
+# github.com/spf13/afero v1.2.2 => github.com/spf13/afero v0.0.0-20160816080757-b28a7effac97
```

Comment:
Out of curiosity, what is `modules.txt` used for?

Reply:
@Matt, all the changes in https://github.com/kubernetes/kubernetes/pull/76518/commits/3f5f9e8c05f9521d81dd1e7d9fe8022cd7619ee6 was generated automatically after running `hack/pin-dependency.sh` and ` hack/update-vendor.sh` (following the instruction in https://github.com/kubernetes/kubernetes/blob/master/go.mod).